packages/user-operation-controller/src/UserOperationController.test.ts

```diff
@@ -0,0 +1,450 @@
+import { ADDRESS_ZERO, EMPTY_BYTES, ENTRYPOINT } from './constants';
+import { Bundler } from './helpers/Bundler';
+import {
+  UserOperationStatus,
+  type PrepareUserOperationResponse,
+  type SignUserOperationResponse,
+  type SmartContractAccount,
+  type UpdateUserOperationResponse,
+} from './types';
+import type { UserOperationControllerMessenger } from './UserOperationController';
+import { UserOperationController } from './UserOperationController';
+import {
+  validateAddUserOperationRequest,
+  validateAddUserOperatioOptions,
```

Comment:
I think it was meant to be `validateAddUserOperationOptions`

Reply:
Done.